mkdocs.yml

```diff
@@ -4,6 +4,8 @@ pages:
   - [styleguide.md, Style Guide]
   - [gettingstarted/index.md, Getting Started, Home]
   - [gettingstarted/structuring.md, Getting Started, Structuring Your Mod]
+  - [blockstates/introduction.md, Models & Rendering, Blockstates]
+  - [blockstates/forgeBlockstates.md, Models & Rendering, Blockstates, Forges Blockstates]
```

Comment:
I take that back, I don't think this is what was causing the build to fail. It seems it's the link at the bottom...